src/screens/Ticketing/Purchase/Overview/components/PurchaseMessages.tsx

```diff
@@ -0,0 +1,103 @@
+import MessageBox from '@atb/components/message-box';
+import {
+  useHasEnabledMobileToken,
+  useMobileTokenContextState,
+} from '@atb/mobile-token/MobileTokenContext';
+import {
+  PreassignedFareProduct,
+  PreassignedFareProductType,
+} from '@atb/reference-data/types';
+import {StyleSheet} from '@atb/theme';
+import {useTicketState} from '@atb/tickets';
+import {PurchaseOverviewTexts, useTranslation} from '@atb/translations';
+import React from 'react';
+import {getOtherDeviceIsInspectableWarning} from '../../../Ticket/utils';
+import {getTrainTicketNoticeText} from '../../../utils';
+import {TariffZoneWithMetadata} from '../../TariffZones';
+
+export type PurchaseWarningsProps = {
+  preassignedFareProduct: PreassignedFareProduct;
+  fromTariffZone: TariffZoneWithMetadata;
+  toTariffZone: TariffZoneWithMetadata;
+};
+
+const PurchaseMessages: React.FC<PurchaseWarningsProps> = ({
+  preassignedFareProduct,
```

Comment:
Do we need to pass complete fare product here? maybe just pass the preassignedFareProduct type instead?